web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -161,6 +161,19 @@ details[open] {
   margin-left: 0;
 }
 
+/**
+Find some way of detecting these base styles in this project theme.
```

Comment:
How about creating custom properties for succes, warning, danger and info, that the base can inherit? They could also be useful when theming alerts and other states?

Reply:
Took a different approach